File: Even_Num_in_range.js
// n is the range in b/w which we are finding the even number
let i, n = 100, sum = 0;
console.log("the even numbers in range are:");
for (i = 1; i <= n; i++) {
    if (i % 2 == 0) {
        sum = sum + i;
        process.stdout.write(i + " ")
    }

}
console.log("\nthe sum of even number in range is: %d", sum);

File: Even_Num_in_range.py
# n is the range in b/w which we are finding the even number
n = int(input("enter the number:"))
sum = 0
print("the even numbers in range are:",end='')
for i in range(1,n+1):
    if (i % 2 == 0):
        sum = sum + i
        print(i , " ",end='')

print("\nthe sum of even number in range is: ", sum)